### ISSUE TITLE
Locate Tactic Notation

### ISSUE THREAD

**Issue by Janno** [coq/coq#6044]
#### Version
Coq 8.7 (and previous)

#### Operating system
Linux but irrelevant.

#### Description of the problem
As far as I can tell (and to the extend of @ppedrot's knowledge), Coq does not have command to locate `Tactic Notation`. This would be quite helpful to have for debugging.

**Comment by samuelgruetter** [comment]
I would like to have this feature, too!

**Comment by maximedenes** [comment]
This issue is worse than it seems: it means that `Locate` sometimes return incorrect information when a tactic notation masks a tactic...

**Comment by bcip** [comment]
I agree with [maximedenes](https://github.com/coq/coq/issues/6044#issuecomment-496903698) that Locate should not include tactics. But there should be a way to locate a tactic notation. How about "Locate Tactic Notation"?
